kitty/data-types.h

```diff
@@ -195,11 +195,10 @@ typedef struct {
     PyObject_HEAD
 
     bool bold, italic, reverse, strikethrough, blink, dim;
-    unsigned int x, y;
+    index_type x, y, last_x, last_y;
```

Comment:
These should be named last_rendered_X/y to make it clear what their purpose is.

Reply:
And these should really be variables in the screen struct rather than the Cursor struct. Cursor objects can come and go (see for example screen_save_cursor). In fact, the information of where the cursor was last rendered on the screen logically belongs to the screen, notth ecursor.

Reply:
👍